### PR TITLE
Fixed a number of issues before public announcement

### PR DESCRIPTION
**Pull request by melissawm**
- Fixed README and index.md wording
- Moved images for consistency across binder and html site
- Added alt text where appropriate
- ~Removed workaround for gh-63~

### CODE REVIEW

File: content/tutorial-deep-learning-on-mnist.md [hunk 1]
```diff
@@ -72,20 +72,6 @@ data_sources = {
...
-# copies of the datasets to prevent unnecessary server requests
-headers = {
-    "User-Agent": "Mozilla/5.0 (X11; Linux x86_64; rv:10.0) Gecko/20100101 Firefox/10.0"
-}
-request_opts = {
-    "headers": headers,
-    "params": {"raw": "true"},
-}
-```
-
```
Comment (rossbar) [inline content/tutorial-deep-learning-on-mnist.md:88]: This isn't so much a workaround as necessary setup for downloading the data from an external source - it needs to be in place in order for the downloading to work (either on binder or for users running locally).

Comment (melissawm) [reply content/tutorial-deep-learning-on-mnist.md:88]: Ah sorry, I misunderstood this!

File: content/tutorial-deep-learning-on-mnist.md [hunk 2]
```diff
@@ -74,7 +74,6 @@ download it.
 
 ```{code-cell} ipython3
 :tags: [remove-cell]
-
```
Comment (rossbar) [inline content/tutorial-deep-learning-on-mnist.md:77]: This blank line separates the cell metadata (the tags in this case) from the actual code in the cell. I'm actually not sure if it's *required* by the myst-nb text-based notebook format, but I personally like keeping them separated by a blank line.

Alternatively, I believe the format supports a different format to make this separation more explicit, something like:
````
```{code-cell} ipython3
---
tags: [remove-cell]
---
# Code starts here
```

**Review by rossbar — COMMENTED** [review]
Generally LGTM - big +1 on moving the non-generated images/gifs/etc. to a static folder. However, by using the `_static` folder inside the `site` directory, the relative links (e.g. `[img](../_static/...)` will only work for the generated site. If a user tries to open a tutorial locally w/ `jupyter notebook`, the images won't be found.

I would recommend creating another `_static` directory in `content` for all of the .pngs and .gifs linked to by the tutorials themselves (`site/_static` contains resources only for the portions of the site that are not meant to be opened in jupyter-notebook, e.g. `index.md`).

I'd be happy to take a stab at that if you'd like, just LMK!

File: content/tutorial-deep-learning-on-mnist.md (hunk 2 above)
Comment (melissawm) [reply content/tutorial-deep-learning-on-mnist.md:77]: Ah I hadn't noticed that - sorry, it was a copy-past mistake!

**Comment by melissawm** [comment]
> I would recommend creating another `_static` directory in `content` for all of the .pngs and .gifs linked to by the tutorials themselves (`site/_static` contains resources only for the portions of the site that are not meant to be opened in jupyter-notebook, e.g. `index.md`).

That's absolutely right and something I overlooked at first, thanks!

**Review by rossbar — APPROVED** [review]
LGTM, thanks @melissawm !

The CI failure is unrelated to these changes.

**Pull request merged**